integration-tests/contract-security-wrong-field-annotations.spec.ts

```diff
@@ -0,0 +1,275 @@
+ *        [a] Value is not a number: undefined
+ *        [valueA] Value is not a number: undefined
+ * 
+ *  With Jakarta there is a change regarding Annotations: https://tezos.gitlab.io/protocols/013_jakarta.html#michelson. 
+ *  This means the sub testcases 3 & 7 in TC-009 will behave differently in Jakarta.
+ *        
+ */
+
+CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
+  const Tezos = lib;
+  const mondaynet = protocol === Protocols.ProtoALpha ? test: test.skip;
+
+  describe(`Test contracts to verify wrong field annotations are leading to failed transactions using: ${rpc}`, () => {
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    });
+
+    mondaynet('Verify annotation combinations on ithacanet', async () => {
+      const addition = await Tezos.contract.originate({
+        code: securityWrongAnnotations,
+        init: `0`,
+      });
+      await addition.confirmation();
+      const contractAddition = await addition.contract();
+
+      //Testcase #1
+      //Calls entrypoint: (pair %add (nat %valueA) (nat %valueB))
+      //Calling Michelson code: CONTRACT %add (pair (nat %valueA) (nat %valueB)) ;
+
+      const op1 = await Tezos.contract
+        .at(contractAddition.address)
+        .then((contract) => {
+          return contract.methodsObject
+            .call({
+              entrypoint: 'add',
+              param: {
+                valueA: `1`,
+                valueB: `2`,
+              },
+            })
+            .send();
+        })
+        .then((op1) => {
+          return op1.confirmation().then(() => op1.hash);
+        });
+
+      //Testcase #2
+      // Calls entrypoint: (pair %addNoAnnot nat nat))
+      // Calling Michelson code: CONTRACT %addNoAnnot (pair (nat %valueA) (nat %valueB)) ;
+      try {
+        const op2 = await Tezos.contract
+          .at(contractAddition.address)
+          .then((contract) => {
+            return contract.methodsObject
+              .call({
+                entrypoint: 'addNoAnnot',
+                param: {
+                  0: `1`,
+                  1: `2`,
+                },
+              })
+              .send();
+          })
+          .then((op2) => {
+            console.log(`Awaiting for ${op2.hash} to be confirmed...`);
+            return op2.confirmation().then(() => op2.hash);
+          });
+      } catch (error: any) {
+        expect(error.message).toContain('[valueA] Value is not a number: undefined');
+      }
+
+      //Testcase #3
+      // Calls entrypoint: (pair %addWrongAnnot (nat %a) (nat %b))
+      // Calling Michelson code: CONTRACT %addWrongAnnot (pair (nat %valueA) (nat %valueB)) ;
+      
+      try{
+      const op3 = await Tezos.contract
+        .at(contractAddition.address)
+        .then((contract) => {
+          return contract.methodsObject
+            .call({
+              entrypoint: 'AddWrongAnnot',
+              param: {
+                a: `1`,
```

Comment:
Instead of a , the annot should be valueA
Instead of b, the annot should be valueB

Reply:
I made the change but it still gives the same error,.